pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu

```diff
@@ -71,17 +76,45 @@ RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86
  && apt-get install -f -y \
  && rm libcublas10_${CUBLAS_VERSION}.deb
 
-RUN wget https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-$OPEN_MPI_VERSION.tar.gz \
- && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
- && cd openmpi-$OPEN_MPI_VERSION \
- && ./configure --prefix=/home/.openmpi \
- && make all install \
- && cd .. \
- && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
- && rm -rf openmpi-$OPEN_MPI_VERSION
-
-ENV PATH="$PATH:/home/.openmpi/bin"
-ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+# RUN wget https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-$OPEN_MPI_VERSION.tar.gz \
+#  && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
+#  && cd openmpi-$OPEN_MPI_VERSION \
+#  && ./configure --prefix=/home/.openmpi \
+#  && make all install \
+#  && cd .. \
+#  && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
+#  && rm -rf openmpi-$OPEN_MPI_VERSION
+
+# ENV PATH="$PATH:/home/.openmpi/bin"
+# ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
```

Comment:
Can this be removed, since it is already being performed at line 105 and 106?

Reply:
Yes hence commented!